docs/self-hosted/kubernetes-troubleshooting.md

```diff
@@ -60,4 +60,21 @@ proxy_ssl_server_name on;
 If you are using different hostnames on the upstream and on the reverse-proxy, set the value in the
 `proxy_ssl_name` directive to the corresponding hostname of the upstream server.
 
+#### Issues with missing package versions after migrating to Kubernetes
```

Comment:
Shouldn't the trouble shoot entry be about what to do if the Private Packagist domains changes?

Reply:
Ah, I understood this to be only relevant for migrations. I'll change this to focus on changing hostnames then.